client/my-sites/theme/main.jsx

```diff
@@ -310,16 +320,22 @@ const ThemeSheet = React.createClass( {
 	},
 
 	renderFeaturesCard() {
-		const { siteSlug, taxonomies } = this.props;
-		const themeFeatures = taxonomies && taxonomies.theme_feature instanceof Array
-		? taxonomies.theme_feature.map( function( item ) {
+		const { isJetpack, siteSlug, taxonomies } = this.props;
+		if ( ! taxonomies || ! isArray( taxonomies.theme_feature ) ) {
+			return null;
+		}
+
+		const themeFeatures = taxonomies.theme_feature.map( function( item ) {
 			const term = isValidTerm( item.slug ) ? item.slug : `feature:${ item.slug }`;
 			return (
 				<li key={ 'theme-features-item-' + item.slug }>
-					<a href={ `/design/filter/${ term }/${ siteSlug || '' }` }>{ item.name }</a>
+					{ isJetpack
+						? <a>{ item.name }</a>
```

Comment:
Just curious if adding the `<a>` is actually doing anything here. Are styles applied to it because its an anchor?
Maybe it could just be `{ item.name }`